docs/what.rst

```diff
@@ -0,0 +1,29 @@
+=====================
+What is a Certificate?
+=====================
+
+A public key or digital *certificate* (formerly called an SSL certificate) uses a public key 
+and a private key to enable secure communication between a client program (web browser, email client, 
+etc.) and a server over an encrypted SSL (secure socket layer) or TLS (transport layer security) connection.
+The certificate is used both to encrypt the communication and to identify the server. The certificate
```

Comment:
-The certificate is used both to encrypt the communication and to identify the server. The certificate
+The certificate is used both to encrypt _the initial stage of_ the communication _(secure key exchange)_ and to identify the server. The certificate


(I believe this is more accurate, isn't it? And I know that I proposed the previous version myself... But I just realized that it was somewhat inaccurate.)